bundles/plugins/org.bonitasoft.studio.actors/src/org/bonitasoft/studio/actors/ui/wizard/page/OrganizationLabelProvider.java

```diff
@@ -68,6 +70,12 @@ public void update(ViewerCell cell) {
 		cell.setStyleRanges(styledString.getStyleRanges());
 	}
 
+    private String getActiveOrganization() {
+        final IScopeContext projectScope = RepositoryManager.getInstance().getCurrentRepository().getScopeContext();
+        return projectScope.getNode(ActorsPlugin.PLUGIN_ID).get(ActorsPreferenceConstants.DEFAULT_ORGANIZATION,
```

Comment:
Sure that the first getNode never return null? If no, check for null and return the ActorsPreferenceConstants.DEFAULT_ORGANIZATION_NAME